sphinxval/utils/metrics.py

```diff
@@ -1110,6 +1110,57 @@ def check_SEDS(h, f, m, n):
         seds = chk - 1
         return seds
 
+# F_beta Score
+def calc_fbeta(h, m, f):
+    """
+    Inputs
+    h = hits
+    m = misses
+    f = false alarms
+
+
+    Formula
+    -------
+    F_beta = ((1+beta^2)*h)/((1+beta^2)*h+f+m*beta^2)
+    beta = 0.5, 1, 2
+
+    Notes
+    -------
+    F1 Score (Beta = 1) is the harmonic mean of precision and recall
+    When Beta = 2 the recall is weighted higher than precision and when Beta = 0.5 precision is weighted higher.
+    """
+    beta = [0.5, 1, 2]
+    score = {}
```

Comment:
Don't think we should return multiple metrics in one call, as I don't believe this is done elsewhere and a one-function one-metric paradigm is preferred.  Instead, make `calc_fbeta(h, m, f, beta)` and return the one result, then `def calc_f1(h, m, f)`, `def calc_f2(h, m, f)` and `def calc_fhalf(h, m, f)` can call that.

Reply:
I agree with splitting the metric - I figured keeping them as one may be controversial but since they are so closely related people may want them together and if I needed to split them it would be easy. So I've done that - they are now three separate scores.